src/main.tsx

```diff
@@ -1,10 +1,28 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
+import { createBrowserRouter,RouterProvider } from 'react-router-dom';
 
-import App from './App.tsx';
+import App from './App';
+import Detail from './pages/Detail';
+import Issues from './pages/Issues';
+import NotFound from './pages/NotFound';
+import './index.css';
+
+const router = createBrowserRouter([
+  {
+    path: '/',
+    element: <App />,
+    children: [
+      {index: true, element: <Issues/>},
```

Comment:
/issues 랑 같은 것을 렌더링하면 <Navigate /> 로 리다이렉션 시켜도 좋을거 같아요!

```suggestion
  {index: true, element: <Navigate to="/issue" replace />},
```